integration/bin/run-integration.sh

```diff
@@ -50,6 +50,7 @@ docker create \
        -e "COOK_SCHEDULER_URL=${COOK_URL}" \
        ${COOK_MULTICLUSTER_ENV} \
        -v ${INTEGRATION_DIR}:/opt/cook/integration \
+       -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
What does this do?

Reply:
This is to map the docker socket from the localhost into the container (so you can run docker inspect on a container running outside the integration test container)

Added a comment

Reply:
Actually, didn't add a comment. Apparently you can't do that in the middle of multiline bash commands?